go/vt/topo/consultopo/lock.go

```diff
@@ -49,6 +53,40 @@ func (s *Server) Lock(ctx context.Context, dirPath, contents string) (topo.LockD
 		return nil, convertError(err, dirPath)
 	}
 
+	return s.lock(ctx, dirPath, contents)
+}
+
+// TryLock is part of the topo.Conn interface.
+// TryLock provides exactly same functionality as 'Lock', the only difference is
+// it tires its best to be non-blocking call. Non-blocking is the best effort though.
+// If there is already lock exists for dirPath then TryLock
+// unlike Lock will return immediately with error 'lock already exists'.
```

Comment:
Since we have the same comments in the interface, we can delete them here and leave just the first line which says `part of topo.Conn interface`.